### DISCUSSION TITLE
How to add a keyword list

### DISCUSSION THREAD

**Discussion by hauni97** [thuliteio/doks#555] [Q&A]
How do you add a keyword list to easily find certain keywords when searching.

**Comment by h-enk** [comment]
Not exactly sure if/how that's possible. @schnerring, could you be of any help? Thanks!

**Reply by schnerring** [reply]
I'm not exactly sure what you mean with a keyword list. Something you define in the front matter, like tags?

For this to work, we'd probably have to implement [complex documents](https://github.com/nextapps-de/flexsearch#complex-documents) for flex search:

```json
{
  "meta": {
    "tag": "cat",
    "id": 0
  },
  "contents": [
    {
      "body": {
        "title": "some title",
        "footer": "some text"
      },
      "keywords": ["some", "key", "words"]
    },
    {
      "body": {
        "title": "some title",
        "footer": "some text"
      },
      "keywords": ["some", "key", "words"]
    }
  ]
}
```
